egs/audioset/AT/local/generate_audioset_manifest.py

```diff
@@ -0,0 +1,134 @@
+import argparse
```

Comment:
Could you add a README.md and a RESULTS.md for this recipe?

Reply:
By the way, it lacks a file `prepare.sh`.

Reply:
Added